nzedb/IRCClient.php

```diff
@@ -580,7 +582,7 @@ protected function _readSocket()
 	protected function _writeSocket($command)
 	{
 		$command .= "\r\n";
-		for ($written = 0; $written < strlen($command); $written += $fWrite) {
+		for ($written = 0, $writtenMax = strlen($command); $written < $writtenMax; $written += $fWrite) {
```

Comment:
Nice optimisation, but I'd put the '$writtenMax = strlen($command)' outside the for on the line before, to make it easier reading